cfgov/teachers_digital_platform/tests/test_forms.py

```diff
@@ -1,7 +1,33 @@
 from django import forms
+from django.core import signing
 from django.test import TestCase
 
-from ..forms import SurveyForm, markup
+from ..forms import SharedUrlForm, SurveyForm, markup
+from ..UrlEncoder import UrlEncoder
```

Comment:
We prefer absolute imports in Python:

```suggestion
from teachers_digital_platform.forms import SharedUrlForm, SurveyForm, markup
from teachers_digital_platform.UrlEncoder import UrlEncoder
```